test/server/cards/03_TWI/units/EncouragingLeadership.spec.ts

```diff
@@ -0,0 +1,40 @@
+describe('EncouragingLeadership', function () {
+    integration(function (contextRef) {
+        describe('EncouragingLeadership\'s ability', function () {
+            it('Gives +1/+1 for this phase to allies', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['encouraging-leadership'],
+                        spaceArena: ['green-squadron-awing'],
+                        groundArena: ['duchesss-champion', 'atst']
+                    },
+                    player2: {
+                        spaceArena: ['star-wing-scout'],
+                        groundArena: ['specforce-soldier'],
+                    },
+                });
+                const { context } = contextRef;
+
+                // Check if effect is applied to both units
+                context.player1.clickCard(context.encouragingLeadership);
+                expect(context.encouragingLeadership).toBeInZone('discard');
+                expect(context.atst.getPower()).toEqual(7);
+                expect(context.atst.getHp()).toEqual(8);
+                expect(context.greenSquadronAwing.getPower()).toEqual(2);
```

Comment:
Need to check the opponent's units to make sure they aren't affected. Also, there isn't a test for a-wing's hp currently